dashboard/config/environments/test.rb

```diff
@@ -75,5 +75,16 @@
   # Set to :debug to see everything in the log.
   config.log_level = :info
 
+  if CDO.running_web_application?
+    # Use default logging formatter so that PID and timestamp are not suppressed.
+    config.log_formatter = Logger::Formatter.new
+
+    # Log condensed lines to syslog for centralized logging.
+    config.lograge.enabled = true
+    config.lograge.formatter = Lograge::Formatters::Cee.new
+    require 'syslog/logger'
+    config.logger = Syslog::Logger.new 'dashboard', Syslog::LOG_LOCAL0
+  end
```

Comment:
similar logging config to other environments, but limited to running web app to make sure it does not get set in unit tests.

Reply:
👍 in particular, very similar to the config for production.rb